src/Shift-Changes.package/ShClassSlotChangeDetector.class/instance/compareSlotCollection.with..st

```diff
@@ -0,0 +1,9 @@
+changes
+compareSlotCollection: a with: b
+	(a size = b size) ifFalse:[^false].
+
+	a with: b do: [ :aSlot :bSlot | 
+			(aSlot name = bSlot name and: [ aSlot species = bSlot species ])
```

Comment:
this will fail for more complex slots like relation slots that have the same species, same name but can be different it slot definition

Reply:
exactly the same method is in the class ClassSlotChnageDetector, should be moved to a superclass and called from here